### ISSUE TITLE
if there are two or more GvrDropdown, an error occurs in the second and subsequent GvrDropdown.

### ISSUE THREAD

**Issue by iwahara** [googlevr/gvr-unity-sdk#523]
If there are two or more GvrDropdown in the scene, Nothing is displayed on click the second and subsequent GvrDropdown.
Then, the following appeared in console.

```
Assertion failed on expression: 'IsFinite(outDistanceForSort)'

transform.position assign attempt for 'Reticle' is not valid. Input position is { NaN, NaN, NaN }.
UnityEngine.Transform:set_position(Vector3)
GvrLaserPointerImpl:OnUpdate() (at Assets/GoogleVR/Scripts/UI/GvrLaserPointerImpl.cs:156)
GvrLaserPointer:LateUpdate() (at Assets/GoogleVR/Scripts/UI/GvrLaserPointer.cs:65)

Invalid AABB a

Invalid AABB a

Assertion failed on expression: 'curveT >= GetRange().first && curveT <= GetRange().second'

Assertion failed on expression: 'curveT >= m_Curve[lhs].time && curveT <= m_Curve[rhs].time'

Assertion failed on expression: 'IsFinite(outDistanceAlongView)'
```

The first GvrDropdown is correct for me.
This problem happens in "Unity 5.6.0f3", but not "Unity 5.4".

The project structure is this image.

<img width="1552" alt="0029-04-04 15 15 06" src="https://cloud.githubusercontent.com/assets/3982522/24643628/2643b18e-194a-11e7-9406-edc0835dec12.png">

Please tell me how to solve the problem as soon as possible.

Regards

**Comment by chaosemer** [comment]
Thanks for the bug report, I've added it to our internal tracker.

**Comment by iwahara** [comment]
This problem resolved by update Unity to ver 5.6.1f1.

thanks.

**Comment by nathanmartz** [comment]
Glad you got it fixed.
